platform/pjsip/templates/stm32f7cube/mods.config

```diff
@@ -5,12 +5,16 @@ configuration conf {
 	@Runlevel(0) include embox.arch.system(core_freq=216000000)
 	@Runlevel(0) include embox.arch.arm.cortexm3.bundle
 	@Runlevel(0) include third_party.bsp.stmf7cube.arch
-	//include embox.arch.arm.vfork
 
 	@Runlevel(0) include embox.kernel.stack(stack_size=10200,alignment=4)
 
-	@Runlevel(0) include embox.arch.arm.armmlib.interrupt(
-							nvic_use_prio=true, nvic_prio_shift=5)
+	//@Runlevel(0) include embox.arch.arm.armmlib.interrupt(
```

Comment:
Uncomment or remove ?
May be move this to templates/arm/stm32f7cube/mods.config?